libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -39,10 +39,7 @@
         </option>
       </select>
     </sky-input-box>
-    <sky-form-errors
-      *ngIf="labelText && hostControl?.errors?.['skyDateRange']"
-      [labelText]="labelText"
-    >
+    <sky-form-errors [labelText]="labelText">
```

Comment:
We still only want to show `sky-form-errors` if `labelText` is defined since its feature is gated by the "easy mode" stuff.

Reply:
I would confirm that using the `async` pipe on the `labelText` input still shows the `sky-form-errors` component. For example, the following should still show the `sky-form-errors` element:
```
<sky-date-range-picker [labelText]="'foobar_string' | skyAppResources"> ...
```

Reply:
Based on the construction of the `sky-form-errors` component and its usage in `sky-input-box` this was always the intended implementation.

A playground test hooking this input up to an observable still allows the errors component to be visible even though the `labelText` isn't set yet.

Reply:
That was a recent change (late last week) and might be a bug. Let's chat offline.